docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+
+Save and close the file. Next, you’ll need to activate this new configuration. Nginx’s convention is to create symbolic links (like shortcuts) from files in `sites-available/` to another folder called `sites-enabled/` as you decide to enable or disable them. Using full paths for clarity, make that link:
+
+```bash
+sudo ln -s /etc/nginx/sites-available/temporal-ui /etc/nginx/sites-enabled/temporal-ui
+```
+
+By default, Nginx includes another configuration file at `/etc/nginx/sites-available/default`, linked to `/etc/nginx/sites-enabled/default`, which also serves its default index page. You’ll need to disable that rule by removing it from `/sites-enabled`, because it conflicts with your new Temporal configuration:
+
+```bash
+sudo rm /etc/nginx/sites-enabled/default
+```
+
+Before reloading Nginx with this new configuration, you'll create another reverse proxy for gRPC API connections to Temporal itself. Create another Nginx configuration at `/etc/nginx/sites-available/temporal`:
+
+```bash
+sudo vim /etc/nginx/sites-available/temporal
+```
+
+Paste the following into the new configuration file, being sure to replace `your_domain` with your domain name.
+
+```
+server {
+    listen 7233;
+    listen [::]:7233;
+    server_name your_domain
+
+    http2 on;
+
+    location / {
+        grpc_pass localhost:7233;
+        allow your_client_ip_address;
+        deny all;
+    }
+}
+```
+
+This configuration is shorter than the previous one, because Nginx only needs to use the `grpc_pass` directive to send gRPC traffic to the server using the same external port (7233) that Temporal's gRPC API runs on. Again, don't forget the the `allow your_client_ip_address;` line. In this case, you'll need an `allow` statement or IP range for everywhere that you plan to run your Temporal Workers, or any other Temporal Client, or connect via the `temporal` CLI.
+
+Save and close the file, and create a symbolic link as before:
+
+```bash
+sudo ln -s /etc/nginx/sites-available/temporal /etc/nginx/sites-enabled/temporal
+```
+
+Now you can restart your Nginx service, so it will reflect your new configuration:
+
+```bash
+sudo systemctl restart nginx
+```
+
+Navigate to your_domain in a web browser, and you should receive the Temporal Web UI. At this point, you're finished with configuration. In the final step, you'll review the logs generated by your Temporal Service, as well as your options for connecting to it from the Temporal CLI or SDK.
+
+## Logging, Restarting, and Connecting to the Temporal Service
```

Comment:
Since each of these points are _super_ short, it feels odd to distinctly call them out. If they were h3s with more content I think it would be more appropriate, but since they're a sentence I think Interacting summarizes them better. 

```suggestion
## Interacting with the Temporal Service
```